src/rules/objectLiteralShorthandRule.ts

```diff
@@ -78,3 +139,10 @@ function handleLonghandMethod(name: ts.PropertyName, initializer: ts.FunctionExp
     }
     return [prefix + sourceFile.text.substring(nameStart, name.end), fix];
 }
+
+function isShorthandAssignment(node: ts.Node): boolean {
+    return (
+        isShorthandPropertyAssignment(node) ||
+        (isMethodDeclaration(node) && node.parent !== undefined ? node.parent.kind === ts.SyntaxKind.ObjectLiteralExpression : false)
```

Comment:
`node.parent` cannot be undefined
also no need for the conditional expression, just use `&&`